CRM/Contact/Form/Search.php

```diff
@@ -326,25 +337,15 @@ public static function getModeSelect() {
    */
   public function buildTaskList() {
     if ($this->_context !== 'amtg') {
```

Comment:
If you figure out what amtg stands for & when it arises please comment the code. My best guess at the moment is that it is 'add member to group'

Reply:
@eileenmcnaughton i think your right as per https://github.com/civicrm/civicrm-core/blob/a4aae59c25dd5823d17b107d3741925a343b0276/CRM/Contact/Form/Search.php#L180

Reply:
Yes, the search classes have a few of these strings - see https://github.com/civicrm/civicrm-core/blob/master/CRM/Contact/Form/Search.php#L176

Reply:
Anyway, I've inserted a comment just above that line